remappers/kubernetesmetrics/kubelet.go

```diff
@@ -38,13 +38,19 @@ func addKubeletMetrics(
 	//pod
 	for i := 0; i < src.Len(); i++ {
 		metric := src.At(i)
-		// kubernetes.pod.cpu.usage.node.pct and kubernetes.pod.memory.usage.node.pct still needs to be implemented
+		// kubernetes.pod.memory.usage.node.pct still needs to be implemented
 		if metric.Name() == "k8s.pod.cpu_limit_utilization" {
 			dp := metric.Gauge().DataPoints().At(0)
 			if timestamp == 0 {
 				timestamp = dp.Timestamp()
 			}
 			cpu_limit_utilization = dp.DoubleValue()
+		} else if metric.Name() == "k8s.pod.cpu.node.utilization" {
```

Comment:
Can we add test for this metric ?

Reply:
it was addressed in https://github.com/elastic/opentelemetry-lib/pull/31, this PR can be closed

Reply:
closing as duplicate